File: A. Vanya and Fence/dart/vanyaandfence/bin/vanyaandfence.dart
import 'dart:io';

int main() {
  int hs, hf, fn = 0, sum = 0, i;
  print('Enter Your frind No.:- ');
  fn = int.parse(stdin.readLineSync());
  print('Enter Hight of Siag.:- ');
  hs = int.parse(stdin.readLineSync());
  for (i = 1; i <= fn; i++) {
    print('Enter your $i hight:- ');
    hf = int.parse(stdin.readLineSync());
    (hf > hs) ? sum++ : null;
    sum++;
  }
  print("$sum");
  return 0;
}
/*
###################################
###################################
####### More code Lost Time #######
###################################
###################################
*/
// import 'dart:io';
// int main() {
//   var ns, ss, hof;
//   int f = 0, c = 0, fn = 0, hs = 0;
//   List<int> hight, row;
//   hight = [];
//   row = [];
//   print('Enter Your frind No.:- ');
//   ns = stdin.readLineSync();
//   print('Enter Hight of Siag.:- ');
//   ss = stdin.readLineSync();
//   fn = int.parse(ns);
//   hs = int.parse(ss);
//   for (int z = 1; z <= fn; z++) {
//     print('Enter your $z hight:- ');
//     hof = stdin.readLineSync();
//     c = int.parse(hof);
//     hight.insert(z - 1, c);
//   }
//   for (int s = 0; s < hight.length; s++) {
//     if (hight[s] > hs) {
//       row.insert(s, 2);
//     } else {
//       row.insert(s, 1);
//     }
//     f = row[s] + f;
//   }
//   print(f);
//   return f;
// }

File: A. Vanya and Fence/pyton/vanyaandFence.py
print("Enter Your frind No.:- ")
ns = int(input())  # No. of his friend
print("Enter Hight of Siag.:- ")
ss = int(input())  # Hight of Siag
i ,Sum = 0 , 0 #Hight of frinds and sum of row

while i <=ns:
        print(f"Enter your {i+1} hight:-  ")
        hs = int(input())
        i+=1
        if hs > ss:
            Sum+=1
        Sum+=1    
        if i == ns :
            break 
print(Sum)

###################################
###################################
####### More code Lost Time #######
###################################
###################################
# print("Enter Your frind No.:- ")
# ns = int(input())  # No. of his friend
# print("Enter Hight of Siag.:- ")
# ss = int(input())  # Hight of Siag
# i = 1
# friend =[]
# row = []
# print(type(ss))
# while i <= ns :
#     print(f"Enter your {i} hight:-  ")
#     hs = int(input())
#     friend.append(hs)
#     i+=1
# for x in friend:
#     if x > ss :
#         p=2
#         row.append(p)

#     else :
#         n=1
#         row.append(n)
# Sum = sum(row)
# print(Sum)